cli/bench/cache_api.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+const cacheName = "cache-v1";
+const cache = await caches.open(cacheName);
+const req = "https://deno.com";
+
+Deno.bench(
+  `cache_storage_open`,
+  { n: 5e2 },
```

Comment:
These do nothing (we run benches as long as there are statistically significant variations between runs). Suggest to remove all of these